integreat_cms/static/src/js/forms/tinymce-init.ts

```diff
@@ -157,19 +161,18 @@ window.addEventListener("load", () => {
       content_css: tinymceConfig.getAttribute("data-content-css"),
       content_style: tinymceConfig.getAttribute("data-content-style"),
       language: tinymceConfig.getAttribute("data-language"),
-      directionality: tinymceConfig.getAttribute("data-directionality") as 'ltr' | 'rtl',
-      element_format: 'html',
+      directionality: tinymceConfig.getAttribute("data-directionality") as
+        | "ltr"
+        | "rtl",
+      element_format: "html",
       setup: (editor: Editor) => {
         addIcon(editor, tinymceConfig, "pin");
         addIcon(editor, tinymceConfig, "www");
         addIcon(editor, tinymceConfig, "email");
         addIcon(editor, tinymceConfig, "call");
         addIcon(editor, tinymceConfig, "clock");
         addIcon(editor, tinymceConfig, "idea");
-        editor.ui.registry.addIcon(
-          "no_translate",
-          parseSvg(require("../../svg/no_translate.svg").default)
-        );
+        addNoTranslateIcon(editor);
```

Comment:
```suggestion
        editor.ui.registry.addIcon(
          "notranslate",
          parseSvg(require("../../svg/notranslate.svg"))
        );
```

I see no reason why we should add a new function for something that's only used once... And I'd use the same name for the tinymce icon and the file name, so either `no_translate` or `notranslate` in both cases.